check_log_ng.py

```diff
@@ -696,6 +702,11 @@ def _check_log_multi(self, logfile_pattern, remove_seekfile=False, tag=''):
                 self._remove_old_seekfile_with_inode(logfile_pattern, tag)
             else:
                 self._remove_old_seekfile(logfile_pattern, tag)
+
+        # if set error_no_exist, return UNKNOWN.
+        if self.config['error_no_exist'] and log_exist == False:
+            self.message = "UNKNOWN: Logfile is not found."
```

Comment:
I think it's better to contain logfile path in output.

Reply:
OK. I modified to display logfile path.